### ISSUE TITLE
Consider adding no-floating-promises

### ISSUE THREAD

**Issue by mxschmitt** [playwright-community/eslint-plugin-playwright#20]
When using TypeScript, this is a great rule to prevent you from forgetting an await statement. For JavaScript its not available afaik.

Reference: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-floating-promises.md

**Comment by mskelton** [comment]
@mxschmitt Is this something you would envision as a rule we enable by default?  The one thing we should be careful with here is forcing users to have a plugin installed that they may not need.  This actually was the case with the jest-playwright config in this library since it required enabling `eslint-plugin-jest`.

One approach we could take would be to have a typescript config that can be extended to enable TS specific rules that make sense for PW.  Something like this:

```js
extends: [
  'plugin:playwright/playwright-test',
  'plugin:playwright/typescript',
]
```
